scripts/bert/dataset.py

```diff
@@ -52,6 +59,350 @@ def get_labels():
+
+    @staticmethod
+    def get_labels():
+        """Get classification label ids of the dataset."""
+        return ['not_entailment', 'entailment']
+
+    @staticmethod
+    def get_metric():
+        """Get metrics Accuracy"""
+        return Accuracy()
+
+
+
+@register(segment=['train', 'dev', 'test'])
+class QNLIDataset(GLUEDataset):
+    """Task class for SQuAD NLI
+
+    Parameters
+    ----------
+    segment : str or list of str, default 'train'
+        Dataset segment. Options are 'train', 'dev', 'test' or their combinations.
+    root : str, default '$GLUE_DIR/QNLI'
+        Path to the folder which stores the QNLI dataset.
+        The datset can be downloaded by the following script:
+        https://gist.github.com/W4ngatang/60c2bdb54d156a41194446737ce03e2e
+    """
+    def __init__(self, segment='train',
+                 root=os.path.join(os.getenv('GLUE_DIR', 'glue_data'), 'RTE')):
+        self._supported_segments = ['train', 'dev', 'test']
+        assert segment in self._supported_segments, 'Unsupported segment: %s'%segment
+        path = os.path.join(root, '%s.tsv'%segment)
+        if segment in ['train', 'dev']:
+            A_IDX, B_IDX, LABEL_IDX = 1, 2, 3
+            fields = [A_IDX, B_IDX, LABEL_IDX]
+        elif segment == 'test':
+            A_IDX, B_IDX = 1, 2
+            fields = [A_IDX, B_IDX]
+        super(QNLIDataset, self).__init__(path, num_discard_samples=1, fields=fields)
+
+    @staticmethod
+    def get_labels():
+        """Get classification label ids of the dataset."""
+        return ['not_entailment', 'entailment']
+
+    @staticmethod
+    def get_metric():
+        """Get metrics Accuracy"""
+        return Accuracy()
+
+
+
+@register(segment=['train', 'dev', 'test'])
+class STSBDataset(GLUEDataset):
+    """Task class for Sentence Textual Similarity Benchmark.
+
+    Parameters
+    ----------
+    segment : str or list of str, default 'train'
+        Dataset segment. Options are 'train', 'dev', 'test' or their combinations.
+    root : str, default '$GLUE_DIR/STS-B'
+        Path to the folder which stores the STS dataset.
+        The datset can be downloaded by the following script:
+        https://gist.github.com/W4ngatang/60c2bdb54d156a41194446737ce03e2e
+    """
+    def __init__(self, segment='train',
+                 root=os.path.join(os.getenv('GLUE_DIR', 'glue_data'), 'STS-B')):
+        self._supported_segments = ['train', 'dev', 'test']
+        assert segment in self._supported_segments, 'Unsupported segment: %s'%segment
+        path = os.path.join(root, '%s.tsv'%segment)
+        if segment in ['train', 'dev']:
+            A_IDX, B_IDX, LABEL_IDX = 7, 8, 9
+            fields = [A_IDX, B_IDX, LABEL_IDX]
+        elif segment == 'test':
+            A_IDX, B_IDX = 7, 8
+            fields = [A_IDX, B_IDX]
+        super(STSBDataset, self).__init__(path, num_discard_samples=1, fields=fields)
+
+
```

Comment:
Method definitions inside a class should be surrounded by a single blank line. https://www.python.org/dev/peps/pep-0008/#blank-lines

Reply:
Thanks. yapf fix this.